scripts/package-builder.js

```diff
@@ -160,11 +175,16 @@ export const Basic = Template.bind({});
 const spec = `
 import React from 'react';
 import { render } from '@testing-library/react';
-import ${PACKAGE_UC} from '.';
+
+import {${PACKAGE_UC}} from '.';
 
 describe('packages/${PACKAGE_LC}', () => {
   test('condition', () => {
 
   })
 })
 `;
+
+const types = ``;
```

Comment:
Should we export an empty prop interface?
```ts
export interface ${PACKAGE_UC}Props {}
```

Reply:
For the last bit in the parentheses, do you mean add
```
export { ${PACKAGE_UC}Props } from '.......types';
```
to index.ts?